### ISSUE TITLE
CLion Spell Checker incorrectly reports 'static as a typo.

### ISSUE THREAD

**Issue by smj389** [intellij-rust/intellij-rust#5551]
## Environment

* **IntelliJ Rust plugin version:** 0.2.123.3132-201
* **Rust toolchain version:** 1.46.0-nightly (feb3536eb 2020-06-09) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2020.2 EAP (CL-202.5103.16)
* **Operating system:** Windows 10 10.0

## Problem description
'static is incorrectly reported as a typo.

## Steps to reproduce
```rust
impl<A, F> Future for Task<A, F>
where
    F: Future<Output = A> + Send + Sync + 'static,
{
}
```

The spell checker highlights 'static as having a typo. The ' is required by the Rust language.
